tests/test_cases/test_discovery/test_discovery.py

```diff
@@ -78,25 +78,22 @@ async def test_arr_scope(dut):
     assert dut.arr[1].arr_sub._path == f"{dut._path}.arr[1].arr_sub"
```

Comment:
I thought I had fixed this verilator vpi bug, but this is still showing up as dut.arr_sub